packages/daemon/src/daemon.js

```diff
@@ -1007,41 +1007,38 @@ const makeDaemonCore = async (
     );
   };
 
-  /** @type {import('./types.js').DaemonCore['incarnateGuest']} */
-  const incarnateGuest = async (hostFormulaIdentifier, deferredTasks) => {
+  /**
+   * Helper for callers of `incarnateNumberedGuest`.
+   * @param {string} hostFormulaIdentifier - The formula identifier of the host.
+   */
+  const incarnateGuestDependencies = async hostFormulaIdentifier =>
```

Comment:
Should be `provideGuestDependencies`. Doesn’t create formulas. Renaming `formulate*` functions should make that distinction clearer.